src/client/actions/user/index.ts

```diff
@@ -232,6 +258,10 @@ const getUrlsForUser = (): ThunkAction<
         .format('D MMM YYYY')
       // eslint-disable-next-line no-param-reassign
       url.editedLongUrl = removeHttpsProtocol(url.longUrl)
+      // eslint-disable-next-line no-param-reassign
```

Comment:
might want to disable the lint for the entire file?

Reply:
I've changed it to disable this particular code block rather than having 4 disable-next-line's